Makefile

```diff
@@ -129,8 +129,12 @@ nodejstest: nodejs
 	npm test
 
 pytest: python
-	cd tools/python_api/test
-	python3 -m pytest -v test_main.py
+ifeq ($(OS),Windows_NT)
+	set PYTHONPATH=tools/python_api/build
+else
+	export PYTHONPATH=tools/python_api/build
+endif
+	python3 -m pytest -v tools/python_api/test
```

Comment:
use `cmake -E env PYTHONPATH=tool/python_api/build python3 ...` and remove the OS-specific conditional :)